ci/deploy_docs.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -e # exit with nonzero exit code if anything fails
```

Comment:
I strongly recommend doctr to simplify the docs deployment.

Reply:
A pull request to the branch is welcome...but at this point I'm using this same procedure on 3 other projects and I don't have the time to figure out doctr.